docs/en/build_and_run.md

```diff
@@ -0,0 +1,162 @@
+
+# Build compilation environment
+
+**Note:**
+1. If you just want to experience the deployment and basic functions of CURVE, **you do not need to compile CURVE**, please refer to [Single-machine deployment](deploy.md).
```

Comment:
The "Single-machine deployment" doc link maybe (deploy_en.md)

Reply:
fix, but about now the deploy_en.md is not accessable.